### DISCUSSION TITLE
standard_init_linux.go:211: exec user process caused "exec format error"

### DISCUSSION THREAD

**Discussion by invictusro** [tiangolo/uwsgi-nginx-flask-docker#364] [Questions] [answered]
The command '/bin/sh -c pip install flask' returned a non-zero code: 1
Hello so I got this error when trying to exec the comand "docker build" because the image is only for Intel arhitecture but I wanted to use it on a raspberry which has ARM arhitecture, is there anything I can do about it ?

**Comment by khoadng** [comment]
@invictusro 
see #67 
basically you have to rebuild the image for ARM by yourself

**Comment by pkuehne** [comment]
Any chance that ARM will be included in the official build pipeline? Now that Dockerhub supports multi-architecture builds?

**Comment by guysoft** [comment]
Hey, I added ARM to a fork of this. Had also to build  tiangolo/uwsgi-nginx-docker.

Docker images here:
https://hub.docker.com/r/guysoft/uwsgi-nginx-flask/

Code here:
https://github.com/guysoft/uwsgi-nginx-flask-docker/

Code for tiangolo/uwsgi-nginx with multiarch pipeline:
https://github.com/guysoft/uwsgi-nginx-docker

Would love to figure out how to PR this.

**Comment by tiangolo** [comment, marked as answer]
I added support to Arm some time ago, but I didn't comment here. :sweat_smile: 

Now you should be able to just run the standard Docker commands. :rocket:
